.circleci/images/ovmf-arm64/Dockerfile

```diff
@@ -0,0 +1,14 @@
+# Copyright 2025 the u-root Authors. All rights reserved
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file.
+
+FROM ubuntu:rolling as base
+
+# Install dependencies
+RUN apt-get update &&                          \
+    apt-get install -y --no-install-recommends \
+        wget
+
+FROM scratch
+RUN wget https://releases.linaro.org/components/kernel/uefi-linaro/16.02/release/qemu64/QEMU_EFI.fd
```

Comment:
Maybe I am overseeing something, but why take it from _linaro_ instead of the [Ubuntu sources](https://ubuntu.com/server/docs/boot-arm64-virtual-machines-on-qemu) the same as for amd64? To me, it looks from the documentation like OVMF/edk2 is shipped with the installation of `qemu-system-arm` (`/usr/share/qemu-efi-aarch64/QEMU_EFI.fd`).

Reply:
Update to use ARM64 OVMF binary QEMU_EFI.fd from Ubuntu qemu-system-arm package. And validated at QEMU env, it works with Linuxboot + U-Root + UPL and target debain ARM64 OS.